tools/RAiDER/delay.py

```diff
@@ -78,6 +78,12 @@ def tropo_delay(
         else:
             height_levels = wm_levels
 
+    if isinstance(zref, str):
+        try:
+            zref = float(zref)
+        except:
+            log.warning('Cannot convert zref={%s} to a number', zref)
+
```

Comment:
```suggestion
    if isinstance(zref, str):
        zref = float(zref)
```

Think built in python error is better than `try`/`except`.

Reply:
this shouldn't be an error; often time it will be None[bool] or 'null'